app-frontend/src/app/pages/library/projects/detail/projectScenes/projectScenes.controller.js

```diff
@@ -176,7 +177,19 @@ export default class ProjectScenesController {
         this.activeModal = this.$uibModal.open({
             component: 'rfPublishModal',
             resolve: {
-                project: () => this.project
+                project: () => this.project,
+                tileTemplateUrl: () => {
+                    let host = this.$location.host();
+                    let protocol = this.$location.protocol();
+
+                    let port = this.$location.port();
+                    let formattedPort = port !== 80 || port !== 443 ? ':' + port : '';
+                    return `${protocol}://${host}${formattedPort}` +
+                        `/tiles/${this.project.organizationId}` +
+                        `/rf_airflow-user` +
+                        `/project/${this.project.id}/{level}/{col}/{row}/` +
+                        `?tag=${(new Date).getTime()}`;
```

Comment:
This works but you may want to have the `()` invoking the date constructor (at least to remove lint warnings). Linter also complains about the backticks instead of single-quotes to represent a string with no interpolation within it. Same thing for the modal settings on the `ProjectEditController`